lemarche/templates/siaes/search_results.html

```diff
@@ -139,6 +139,9 @@
                                         <div class="col-12 col-md-6 col-lg-3" title="{{ form.has_groups.help_text }}">
                                             {% bootstrap_field form.has_groups show_help=False %}
                                         </div>
+                                        <div class="col-12 col-md-6 col-lg-3" title="{{ form.ca.help_text }}">
+                                            {% bootstrap_field form.ca show_help=False %}
```

Comment:
```suggestion
                                            {% bootstrap_field form.ca %}
```
Le `show_help` je l'avais rajouté au-dessus car le form field `has_groups` avait un help_text. là le form field `ca` n'en a pas, donc pas besoin :) (idem pour le title dans le div parent)